kanalony-storage-logic/src/main/scala/kanalony/storage/logic/queries/HourlyUserActivityPrtnEntryClstCountry.scala

```diff
@@ -0,0 +1,49 @@
+package kanalony.storage.logic.queries
+
+import kanalony.storage.generated.hourly_user_activity_prtn_entry_clst_countryRow
+import kanalony.storage.logic._
+import kanalony.storage.logic.queries.model._
+import org.joda.time.DateTime
+
+import scala.concurrent.Future
+
+/**
+ * Created by elad.benedict on 2/11/2016.
+ */
+
+class HourlyUserActivityPrtnEntryClstCountryQuery extends QueryBase[HourlyUserActivityPrtnEntryClstCountryParams, hourly_user_activity_prtn_entry_clst_countryRow] with UserActivityQuery {
+  protected override def extractParams(params: QueryParams): HourlyUserActivityPrtnEntryClstCountryParams = {
+
+    val (partnerIds, entryIds) = QueryParamsValidator.extractEqualityConstraintParams[Int,String]((Dimensions.partner, Dimensions.entry), params)
+    val typedParams = HourlyUserActivityPrtnEntryClstCountryParams(params.start, params.end, partnerIds, entryIds, List(params.metric.id))
+    return typedParams
+  }
+
+  private[logic] override def executeQuery(params: HourlyUserActivityPrtnEntryClstCountryParams): Future[List[hourly_user_activity_prtn_entry_clst_countryRow]] = {
+    val year = List(params.startTime.getYear)
+    val rawQueryResult = dbApi.H_UA_PartnerEntry_Country_StorageClient.query(params.partnerId,
+                              params.entryId,params.metric,year,params.startTime,params.endTime)
+      .fetch()(dbApi.session, scala.concurrent.ExecutionContext.Implicits.global, dbApi.keyspace)
+    rawQueryResult
```

Comment:
I prefer to leave it for readability, as it's not immediately clear what's returned from this 3 line statement